libImaging/JpegDecode.c

```diff
@@ -189,6 +218,10 @@ ImagingJpegDecode(Imaging im, ImagingCodecState state, UINT8* buf, int bytes)
             context->cinfo.out_color_space = JCS_GRAYSCALE;
         else if (strcmp(context->rawmode, "RGB") == 0)
             context->cinfo.out_color_space = JCS_RGB;
+    #ifdef JCS_EXTENSIONS
```

Comment:
This simple check is enough because anything else is checked when `rawmode` is set to `RGBX`.